Tasks/Gradle/task.json

```diff
@@ -227,126 +228,30 @@
             "name": "sqAnalysisEnabled",
             "aliases": [ "sonarQubeRunAnalysis" ],
             "type": "boolean",
-            "label": "Run SonarQube Analysis",
+            "label": "Run SonarQube or SonarCloud Analysis",
             "required": true,
             "defaultValue": "false",
             "groupName": "CodeAnalysis",
-            "helpMarkDown": "Run a [SonarQube analysis](https://go.microsoft.com/fwlink/?LinkID=708598) after executing the current goals. 'install' or 'package' goals should be executed first."
-        },
-        {
-            "name": "sqConnectedServiceName",
-            "aliases": [ "sonarQubeServiceEndpoint" ],
-            "type": "connectedService:Generic",
-            "label": "SonarQube Endpoint",
-            "required": true,
-            "helpMarkDown": "The endpoint that specifies the SonarQube server to use",
-            "groupName": "CodeAnalysis",
-            "visibleRule": "sqAnalysisEnabled = true"
-        },
-        {
-            "name": "sqProjectName",
-            "aliases": [ "sonarQubeProjectName" ],
-            "type": "string",
-            "label": "SonarQube Project Name",
-            "required": true,
-            "helpMarkDown": "The SonarQube project name, i.e. sonar.projectName.",
-            "groupName": "CodeAnalysis",
-            "visibleRule": "sqAnalysisEnabled = true"
-        },
-        {
-            "name": "sqProjectKey",
-            "aliases": [ "sonarQubeProjectKey" ],
-            "type": "string",
-            "label": "SonarQube Project Key",
-            "required": true,
-            "helpMarkDown": "The SonarQube project unique key, i.e. sonar.projectKey.",
-            "groupName": "CodeAnalysis",
-            "visibleRule": "sqAnalysisEnabled = true"
-        },
-        {
-            "name": "sqProjectVersion",
-            "aliases": [ "sonarQubeProjectVersion" ],
-            "type": "string",
-            "label": "SonarQube Project Version",
-            "required": true,
-            "helpMarkDown": "The SonarQube project version, i.e. sonar.projectVersion.",
-            "groupName": "CodeAnalysis",
-            "visibleRule": "sqAnalysisEnabled = true"
+            "helpMarkDown": "Run a [SonarQube or SonarCloud analysis](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html) after executing the current goals. 'install' or 'package' goals should be executed first."
         },
         {
             "name": "sqGradlePluginVersion",
             "aliases": [ "sonarQubeGradlePluginVersion" ],
-            "type": "string",
-            "label": "SonarQube Gradle Plugin Version",
+            "type": "pickList",
```

Comment:
PickList seems like a bad idea for version which might change going forward. You can have a radio button for "Use plugin applied in your build.gradle" or "Specify version number" and default the the specific version number to 2.6.1

Reply:
Good idea!